src/entities/enum/gateway.enum.ts

```diff
@@ -0,0 +1,14 @@
+﻿export enum GatewayPlacement {
+    "NONE" = "NONE",
```

Comment:
Something here doesnt seem right :-)..

Reply:
Get this error when posting: 
![image](https://github.com/OS2iot/OS2IoT-backend/assets/70511721/e33e0351-4e25-4ab0-8d42-1b9aaa34b445)